lib/parser/objects.ts

```diff
@@ -135,6 +135,7 @@ export class OIRange implements Range {
 
 export class ObjectBase {
   lexerToken?: OLexerToken;
+  readonly sourceFile: OFile;
```

Comment:
rootFile? sourceFile seems weird

Reply:
yes, makes more sense. Changed that.
Was just going for a differentiation from `rootElement`